Week-2/fan-page/index.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Metallica Fan Webpage</title>
+    <style>
+      body {
+        font-family: Arial, sans-serif;
+        margin: 40px;
+      }
+      a {
+        color: blue;
+        text-decoration: underline;
+      }
+    </style>
+  </head>
+
+  <body>
+    <h1>Metallica Fan Webpage</h1>
+
+    <p>
+      Metallica is an American heavy metal band that formed in 1981. The band's
+      original members include James Hetfield, Lars Ulrich, and soon after they
+      were joined by Kirk Hammett and Cliff Burton. Over the years, while the
+      lineup changed slightly, their music continued to resonate with fans
+      worldwide, giving them the title of one of the greatest metal bands of all
+      time.
+    </p>
+
+    <p>
+      Visit the
+      <a href="https://www.metallica.com/" target="_blank">official website</a>
```

Comment:
It's a good practice to include rel="noopener noreferrer" attributes in your anchor tags when using target="_blank". This can help improve security and performance. 

For example:<a href="https://www.metallica.com/" target="_blank" rel="noopener noreferrer">official website</a>

Reply:
```html
<a href="https://www.metallica.com/" target="_blank" rel="noopener noreferrer">official website</a>

```